app/controllers/decidim/solutions/solutions_controller.rb

```diff
@@ -9,13 +9,24 @@ class SolutionsController < Decidim::Solutions::ApplicationController
       include FilterResource
       include Paginable
       include OrderableSolutions
+      include Decidim::Sdgs::SdgsHelper
 
       helper Decidim::CheckBoxesTreeHelper
+      helper Decidim::Sdgs::SdgsHelper
 
       helper_method :solutions
 
       def index; end
 
+      def show
+        @solution = Solution.find(params[:id])
```

Comment:
Maybe you could split in some methods the variable assignments, for example as seen here:
- https://github.com/gencat/decidim-challenges/blob/3516a7c4c83e13e49e794e0278a994bd6496fe23/app/controllers/decidim/solutions/admin/solutions_controller.rb#L78